.gitignore

```diff
@@ -1,5 +1,4 @@
 .hermit/
-.vscode/
 !.vscode/extensions.json
```

Comment:
We might want to keep this `.vscode/` line here to avoid `launch.json` and other vscode related files getting committed to git.